project_legal_management/views/lm_resolution_views.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2021 NuoBiT Solutions - Kilian Niubo <kniubo@nuobit.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl) -->
+<odoo>
+    <record id="lm_resolution_form_view" model="ir.ui.view">
+        <field name="name">lm.resolution.form.view</field>
+        <field name="model">lm.resolution</field>
+        <field name="arch" type="xml">
+            <form string="Resolution Form">
+                <sheet>
+                    <div class="oe_title">
+                        <h1>
+                            <field name="name"/>
+                        </h1>
+                    </div>
+                    <group>
+                        <field name="parent_id"/>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record id="lm_resolution_tree" model="ir.ui.view">
+        <field name="name">project.legal.management.lm.resolution.tree</field>
+        <field name="model">lm.resolution</field>
+        <field name="arch" type="xml">
+            <tree string="Issues">
+                <field name="name"/>
+                <field name="parent_id"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="project_resolution_list_action" model="ir.actions.act_window">
+        <field name="name">Resolution</field>
```

Comment:
Same as before, check OCA guidelines https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#252security-view-and-action